src/selection_criteria.rs

```diff
@@ -126,6 +126,30 @@ pub struct ReadPreferenceOptions {
     /// specified for an operation, the operation will return an error.
     #[builder(default)]
     pub max_staleness: Option<Duration>,
+
+    /// Specifies hedging behavior for reads.
+    ///
+    /// See the [MongoDB docs](https://docs.mongodb.com/manual/core/read-preference-hedge-option/) for more details.
+    #[builder(default)]
+    pub hedge: Option<HedgedRead>,
```

Comment:
Based on discussion back when this option was originally added to the spec, I believe the consensus is that we need to be future-compatible with any new options that might be added to the document value under the `hedge` field. To that end, I made a struct called `HedgedRead` (although I welcome feedback on the name, as I'm not super thrilled with anything I came up with) and marked it as non-exhaustive.

Reply:
The best thing I could come up with is `HedgedReadOptions`, but I'm not sure if that's any better since there's only one at the moment.